core/arch/arm/kernel/proc_a32.S

```diff
@@ -39,6 +40,7 @@
 
 /* void cpu_spin_lock(lock address) - lock mutex */
 FUNC cpu_spin_lock , :
+UNWIND(	.fnstart)
```

Comment:
This pattern as well as `UNWIND(.fnend)`/`END_FUNC` is used a number of times, would it make sense to define variants of `FUNC` and `END_FUNC`?